check

```diff
@@ -0,0 +1,13 @@
+monitorEvents($0, 'mouse') //in chrome: to see the mouse events
```

Comment:
What is a `check` file?

Reply:
Oh you can delete it if you want. It is a text file. I just put in some information in there to aid me in case I forget. The first line allows me to monitor mouse events on chrome. Apparently, firefox does not support it. The second line is a workaround for firefox.